src/SFA.DAS.ApplyService.Application.UnitTests/Handlers/Gateway/UpdateGatewayPageAnswerRequestHandlerTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoFixture;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApplyService.Application.Apply;
+using SFA.DAS.ApplyService.Application.Apply.Gateway;
+using SFA.DAS.ApplyService.Application.Interfaces;
+using SFA.DAS.ApplyService.Domain.Audit;
+using SFA.DAS.ApplyService.Domain.Entities;
+
+namespace SFA.DAS.ApplyService.Application.UnitTests.Handlers.Gateway
+{
+    [TestFixture]
+    public class UpdateGatewayPageAnswerRequestHandlerTests
+    {
+        private UpdateGatewayPageAnswerRequestHandler _handler;
+        private Mock<IApplyRepository> _repository;
+        private Mock<IAuditService> _auditService;
+        private Fixture _autoFixture;
+        private Guid _applicationId;
+        private GatewayPageAnswer _existingGatewayPageAnswer;
+        private Domain.Entities.Apply _existingApplication;
+        private string _pageId;
+        private string _userId;
+        private string _userName;
+        private string _status;
+        private string _comments;
+
+
+        [SetUp]
+        public void SetUp()
+        {
+            _autoFixture = new Fixture();
+            _repository = new Mock<IApplyRepository>();
+            _auditService = new Mock<IAuditService>();
+
+            _applicationId = Guid.NewGuid();
+            _pageId = _autoFixture.Create<string>();
+            _userId = _autoFixture.Create<string>();
+            _userName = _autoFixture.Create<string>();
+            _status = _autoFixture.Create<string>();
+            _comments = _autoFixture.Create<string>();
```

Comment:
These are fine, although I'd personally have used realistic values so that they have more meaning when observing the tests